### ISSUE TITLE
Clients sometimes request things that need to be passed through to the underlying solr. See gist for a sample.

### ISSUE THREAD

**Issue by marksteele** [basho/yokozuna#23]
https://gist.github.com/4207723

**Comment by rzezeski** [comment]
Replicating the exception here for posterity.

```
rt_sigaction(SIGALRM, {SIG_DFL, [], SA_RESTORER, 0x7f1a3e8b0920}, NULL, 8) = 0
socket(PF_INET, SOCK_STREAM, IPPROTO_TCP) = 3
fcntl(3, F_GETFL)                       = 0x2 (flags O_RDWR)
fcntl(3, F_SETFL, O_RDWR|O_NONBLOCK)    = 0
connect(3, {sa_family=AF_INET, sin_port=htons(8098), sin_addr=inet_addr("192.168.32.1")}, 16) = -1 EINPROGRESS (Operation now in progress)
poll([{fd=3, events=POLLOUT|POLLWRNORM}], 1, 29999) = 1 ([{fd=3, revents=POLLOUT|POLLWRNORM}])
getsockopt(3, SOL_SOCKET, SO_ERROR, [0], [4]) = 0
sendto(3, "POST /search/select/?version=2.2&indent=on&wt=xml HTTP/1.1\r\nUser-Agent: PHP Solr Client 0.9.11\r\nHost: 192.168.32.1:8098\r\nAccept: */*\r\nAccept-Charset: utf-8\r\nKeep-Alive: 300\r\nConnection: keep-alive\r\nCo"..., 324, MSG_NOSIGNAL, NULL, 0) = 324
poll([{fd=3, events=POLLIN|POLLPRI|POLLRDNORM|POLLRDBAND}], 1, 0) = 1 ([{fd=3, revents=POLLIN|POLLRDNORM}])
poll([{fd=3, events=POLLIN|POLLPRI|POLLRDNORM|POLLRDBAND}], 1, 0) = 1 ([{fd=3, revents=POLLIN|POLLRDNORM}])
recvfrom(3, "HTTP/1.1 405 Method Not Allowed\r\nServer: MochiWeb/1.1 WebMachine/1.9.2 (someone had painted it blue)\r\nDate: Tue, 04 Dec 2012 19:20:53 GMT\r\nContent-Length: 0\r\nAllow: GET\r\n\r\n", 16384, 0, NULL, NULL) = 172
write(2, "PHP Warning:  SolrClient::query():  in /root/riak-php/search.php on line 14\n", 76PHP Warning:  SolrClient::query():  in /root/riak-php/search.php on line 14
) = 76
write(2, "PHP Fatal error:  Uncaught exception 'SolrClientException' with message 'Unsuccessful query request : Response Code 405. (null)' in /root/riak-php/search.php:14\nStack trace:\n#0 /root/riak-php/search.p"..., 304PHP Fatal error:  Uncaught exception 'SolrClientException' with message 'Unsuccessful query request : Response Code 405. (null)' in /root/riak-php/search.php:14
```

**Comment by rzezeski** [comment]
@marksteele Thanks for the report.  Yokozuna doesn't like that the select is run via a `POST` method.  As Yokozuna is meant to provide the same interface as Solr this is a bug.

```
sendto(3, "POST /search/select/?version=2.2&indent=on&wt=xml HTTP/1.1\r\nUser-Agent: PHP Solr Client 
```

**Comment by rzezeski** [comment]
This has been fixed by #27.
